specification/datafactory/resource-manager/Microsoft.DataFactory/2017-09-01-preview/entityTypes/Dataset.json

```diff
@@ -259,54 +275,6 @@
         "modelAsString": true
       }
     },
-    "DatasetPartitionValue": {

```

Comment:
> @gadongji says:
*Remove mistakenly added property "partitionedBy" from ADLS and HTTP dataset

I assume that this block of deletions is what you're referring too. What is the status before this PR? If folks use an instance of this, will requests fail?

Reply:
No, the request will not fail and "PartitionedBy" property will simply be ignored. We are aware that this seems like a breaking change, but still want to make this change for 2 considerations:
1. This is a bug fix more than a breaking change.
2. ADF team already has a client breaking change for this release planned – support for additionalProperties which is priority 1 for them to get in December release.
Please do not reject this PR because of concerns about breaking changes, thanks!

Reply:
Aha, so the reason I worry about breaking changes is because it will literally break peoples' builds! Our team's current recommendation is to leave these fields in, but update the description to make clear that the use of these objects is discouraged or no longer supported.

Reply:
Yes, we know, this conversation has happened internally some time ago. And we choose this time to do this change because there will be other breaking changes in this release as well. We do not want to break peoples' builds twice. Please be aware that this is actually a "bug fix" and have to be done sooner or later. Please do help us make this change into this release, thanks!